### PR TITLE
Don't repeat groupId and version

### PR DESCRIPTION
**Pull request by LorenzoBettini**
which are inherited from the parent pom

Signed-off-by: Lorenzo Bettini <lorenzo.bettini@gmail.com>

### CODE REVIEW

**Comment by cdietrich** [comment]
I remember that this will destroy something. Had done that in the past and had to revert

**Comment by LorenzoBettini** [comment]
really? how can it be. It's standard in poms not to repeat groupId and version from the parent pom, unless they have to be different; in fact you get a warning in Eclipse. @dhuebner ?

**Comment by LorenzoBettini** [comment]
@cdietrich you probably refer to this https://github.com/eclipse/xtext-maven/pull/15 but that was due to the directory structure, if I understand correctly

**Comment by cdietrich** [comment]
It broke with a version change or something like that. Can you check the history? Cannot do that from phone

**Comment by cdietrich** [comment]
See https://github.com/eclipse/xtext-maven/commit/26eebdc70ffee10ca28d83a1428f1ba91fdcf8fb

**Comment by cdietrich** [comment]
Unfortunately miro did not document why. And am not sure if my fix bogus structure change later fixed the real cause

**Comment by cdietrich** [comment]
So for me the publishing does not work. I assume it is typefox publisher plugin that runs on eclipse Jenkins

**Comment by LorenzoBettini** [comment]
What do you mean that publishing does not work?

**Comment by cdietrich** [comment]
there is this https://github.com/TypeFox/publishing
gradle plugin that is used to publish the artifacts to sonatype/maven central.
and it looks like it wants to have an explicit version.
i cannot tell you why.

**Comment by cdietrich** [comment]
https://ci.eclipse.org/xtext/view/Xtext-Xtend/job/xtext-snapshots/687/console

**Comment by cdietrich** [comment]
https://github.com/eclipse/xtext-xtend/blob/master/org.eclipse.xtend.maven.plugin/pom.xml
uses the same pattern.
other maven artifacts we dont deploy with the publisher plugin

**Comment by cdietrich** [comment]
hey i even found 
https://github.com/TypeFox/publishing/issues/1
which i created ;)

**Comment by cdietrich** [comment]
=> if we dont get that publishing plugin fixed i dont see a chance

File: org.eclipse.xtext.maven.plugin/pom.xml [hunk 1]
```diff
@@ -7,7 +7,11 @@
 		<relativePath>../org.eclipse.xtext.maven.parent</relativePath>
 	</parent>
 	<modelVersion>4.0.0</modelVersion>
+	<!-- groupId and version must be repeated for publishing
+		see https://github.com/eclipse/xtext-maven/pull/40#issuecomment-399433375 -->
+	<groupId>org.eclipse.xtext</groupId>
```
Comment (cdietrich) [inline org.eclipse.xtext.maven.plugin/pom.xml]: :+1: can you do the same pr for xtext-xtend

**Comment by LorenzoBettini** [comment]
I left a comment on the pom, just not to forget about that next time ;)
if you think it's still worthwhile to merge the PR just for this comment.

File: org.eclipse.xtext.maven.plugin/pom.xml (hunk 1 above)
Comment (LorenzoBettini) [reply org.eclipse.xtext.maven.plugin/pom.xml]: yes I will
BTW... I hate gradle :D

**Comment by LorenzoBettini** [comment]
yes, I will while merging the PR

**Comment by LorenzoBettini** [comment]
I merged by squashing; unfortunately the main commit message was the original one... hope it's still OK

**Pull request merged**